packages/wpcom.js/src/lib/site.media.js

```diff
@@ -152,6 +153,23 @@ Media.prototype.addFiles = function ( query, files, fn ) {
 		}
 	}
 
+	const videoFiles = [];
+	const isArray = Array.isArray( files );
+	files = isArray ? files : [ files ];
+
+	files = files.filter( ( file ) => {
+		if ( !! file.type && file.type.startsWith( 'video/' ) ) {
+			videoFiles.push( file );
+			return false;
+		}
+		return true;
+	} );
+
+	if ( videoFiles.length ) {
+		const uploader = new TusUploader( this.wpcom, this._sid );
+		return uploader.startUpload( videoFiles );
```

Comment:
Does this do any error handling? What if it fails?

Reply:
As we return a promise, and the promise calls reject on failure, I assumed it would "just work", but I'll actually try to test that in order to confirm

Reply:
Seems to work fine p1648637460816249-slack-C9W0QF6KA